### DISCUSSION TITLE
Fermata not plying my IPTV M3U anymore

### DISCUSSION THREAD

**Discussion by tbreitha** [AndreyPavlenko/Fermata#303] [General]
Version 1.9.4 my imported IPTV list shows up correct incl. EPG and channel icons, but when i want to start streaming, nothing happens. The same list (URL) is working just fine with IPTV Pro app or Emby, only Fermata doesn't want to play the TV Stream.
Any idea what i can try?
regards
Tom

**Comment by tbreitha** [comment]
I now found out, that the VLC video engine seems to be the problem. When i change it to exoplayer it is working for almost all channels, just a few have only audio without picture. I prefer VLC because i can adjust the audio delay and it was working with 1.9.3
Any idea or suggestions what i can do to fix it?

**Reply by AndreyPavlenko** [reply]
Can you send me a stream url that does not work with VLC?

**Comment by tbreitha** [comment]
Hi Andrey,
thanks for replying.
I have a Pixel 7 Pro and got the Android 14 update yesterday.
Since then, Fermata is not working anymore. It freezes as soon as I try to start a stream, doesn’t matter if it is an IPTV Stream or a local movie.
Fermata is my favorite App and I’m very sad it stopped working. Hope you can help me…..

![grafik](https://github.com/AndreyPavlenko/Fermata/assets/100781904/93461139-d8c0-4afe-9e79-743e424d2c48)

My video settings:
![grafik](https://github.com/AndreyPavlenko/Fermata/assets/100781904/4b1c7256-d245-4a77-9553-3982ac7cbdc4)


Von: Andrey Pavlenko ***@***.******@***.***>>
Gesendet: Freitag, 6. Oktober 2023 09:02
An: AndreyPavlenko/Fermata ***@***.******@***.***>>
Cc: tbreitha ***@***.******@***.***>>; Author ***@***.******@***.***>>
Betreff: Re: [AndreyPavlenko/Fermata] Fermata not plying my IPTV M3U anymore (Discussion #303)


Can you send me a stream url that does not work with VLC?

**Reply by AndreyPavlenko** [reply]
Does it hang with VLC only? Have you tried other engines?

**Reply by tbreitha** [reply]
since Android 14 it hangs with every engine. audio is there but no picture and the app is frozen. after a couple seconds a popup appears like in the screenshot: app is not responding.

**Reply by AndreyPavlenko** [reply]
Seems, in Android 14 backward compatibility has been broken.
Try this - https://drive.google.com/file/d/1lYr5T7Hu5DfCvfV3wepajrvRsmjw57d5/view?usp=sharing

**Comment by tbreitha** [comment]
Perfekt, thanks. That fixed it. All working again incl. VLC.

Thank you for the quick support.
Cheers
Tom

________________________________
Von: Andrey Pavlenko ***@***.***>
Gesendet: Freitag, 6. Oktober 2023 23:13
An: AndreyPavlenko/Fermata
Cc: tbreitha; Author
Betreff: Re: [AndreyPavlenko/Fermata] Fermata not plying my IPTV M3U anymore (Discussion #303)


Seems, in Android 14 backward compatibility has been broken.
Try this - https://drive.google.com/file/d/1lYr5T7Hu5DfCvfV3wepajrvRsmjw57d5/view?usp=sharing

**Comment by aamaamilan** [comment]
Hello Andrey
can you help to bring Fermata back to work on Android 14.
it's not appear any more, in the mobile it's working that one you posted in the comment but not in Android Auto.
I have the google P6 Pro.

**Reply by tbreitha** [reply]
Don't think it is an Fermata issue. I'm using a pixel 7 pro with A14 and latest AA and all works fine. 
![Screenshot_20231009-212857](https://github.com/AndreyPavlenko/Fermata/assets/100781904/29bb6201-1e9d-4dec-b59c-fa0316f882a4)
![Screenshot_20231009-213200](https://github.com/AndreyPavlenko/Fermata/assets/100781904/43ce6591-4f67-4bc9-96e1-d43f2435bf8a)
![Screenshot_20231009-213500](https://github.com/AndreyPavlenko/Fermata/assets/100781904/21f95fc5-e32d-424d-8ebe-45d95d0d4b63)

**Reply by aamaamilan** [reply]
have you reinstalled with normal way after the update to A14?

**Reply by tbreitha** [reply]
After the update, it was still there. I only updated Fermata with the new version Andrey posted above.

**Comment by aamaamilan** [comment]
![Screenshot_20231009-213452](https://github.com/AndreyPavlenko/Fermata/assets/81694399/05bc437c-92ab-4428-8e24-e1ec2e537962)

**Reply by tbreitha** [reply]
Do you have the lsposed module active?
![Screenshot_20231009-213930](https://github.com/AndreyPavlenko/Fermata/assets/100781904/341b9f9e-e0a5-4b2c-b05f-7459af02bff7)

**Reply by aamaamilan** [reply]
no, how can I get this?

**Reply by tbreitha** [reply]
Root with Magisk and install lsposed module

**Reply by aamaamilan** [reply]
ok, so the root is required, Thanks for the info mate.

**Reply by tbreitha** [reply]
Not sure, if root is required for fermata but I have my pixel rooted since the beginning. All working incl. Banking apps and all other stuff.

**Reply by aamaamilan** [reply]
always rooting is the best way to solve the security issue with google. I will keep try without rooting, if I fix it I will replay here, thanks againg for the info.

**Comment by tbreitha** [comment]
When trying to play a video from Amazon prime video with fermata browser, I'm receiving the following error message:

Webseite nicht verfügbar
Die Website unter intent://watch.amazon.de/watch?asin=B0B6JSKR61&time=0&territory=DE&ref_=atv_dp_atf_prime_sd_mv_play_t1ADAAAAAA0wr0#Intent;scheme=https;package=com.amazon.avod.thirdpartyclient;end konnte nicht geladen, weil:

net::ERR_UNKNOWN_URL_SCHEME

Is there any solution to fix it?

**Reply by AndreyPavlenko** [reply]
It redirects to the amazon prime app.

**Reply by tbreitha** [reply]
ah ok, makes sense. any chance to force playing it in the browser? Some episodes of series are playing just fine and some fail with the above error message.

**Reply by AndreyPavlenko** [reply]
Have you tried to enable the desktop mode?

**Reply by tbreitha** [reply]
where can i set desktop mode? in fermata?

**Reply by AndreyPavlenko** [reply]
There is a menu option, in the bottom right corner.

**Reply by tbreitha** [reply]
OK, that works but the video is cut off, the aspect ratio doesn't fit the screen :-( Video is much too large and can#t find any option to reduse it.

**Reply by tbreitha** [reply]
ok got it. Thank you so much, working now, pitch to zoom is the soltution :-)

**Comment by ChanelManso** [comment]
also by mee, fermata was not playing my M3U playlist any more, I had to disable VLC, but I have other problems.
I have tried Fermata auto from here, installed it and Fermata Control, but my Android 14 (S24 Ultra) don't show it on the AA.
It only shows the version installed from Google Play Store, is this the same?

> <img width="319" alt="Fermata2" src="https://github.com/AndreyPavlenko/Fermata/assets/160826418/c7af19ce-fdba-4e61-ad59-6543b27d5c68">


Only the Fermata Version installed from Google Play Store is showed in AA, and this one plays my M3U list with disabled VLC, but only Audio, I have no video signal on my AA. This version is called only Fermata

What is wrong here? Which version should I use? which one is the right one for me, to get IPTV working on AA?

> <img width="305" alt="Fermata" src="https://github.com/AndreyPavlenko/Fermata/assets/160826418/4d428b8c-1034-4b49-81ac-bef55323bc61">

What can I do to get also the Video, It is a IPTV M3U file.





Do I need to allow AA to use unknown sources?

> <img width="326" alt="Fermata3" src="https://github.com/AndreyPavlenko/Fermata/assets/160826418/4a5afe47-368b-40ab-a14c-6bbdb60ece97">


Thx for your help.

Manuel

**Reply by AndreyPavlenko** [reply]
https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1934257166

**Reply by ChanelManso** [reply]
Used Method B and bought a dongle,  but I still have the same problem as before. 
Only audio on iptv.

**Reply by AndreyPavlenko** [reply]
With AAWireless it definitely works. Confirmed by many users. All you need is to enable the developer mode in the AAWireless settings.

**Reply by ChanelManso** [reply]
It works
